dev/journal_mode.h

```diff
@@ -42,14 +42,14 @@ namespace sqlite_orm {
             std::transform(str.begin(), str.end(), std::back_inserter(upper_str), [](char c) {
                 return static_cast<char>(std::toupper(static_cast<int>(c)));
             });
-            static std::array<journal_mode, 6> all = {
+            static std::array<journal_mode, 6> all = {{
```

Comment:
why?

Reply:
Because it is a compile error with GCC under the stricter modes I am using.
`std::array` does not have a constructor that takes an init-list. C++ standard suggests that the second braces can be elided, but the 'official' way is to use two sets of braces.
See https://stackoverflow.com/questions/8192185/using-stdarray-with-initialization-lists